spec/requests/user_blocks_spec.rb

```diff
@@ -83,8 +83,10 @@
     end
```

Comment:
I think blocker and blocked should be reloaded with `let_it_be_changeable()` at the top, as by destroying the object we are making sure that if we ever switch on the random order, another test won't have the objects we destroyed in another one.

Reply:
An alternative strategy, if recreating the object anew is too costly, is to create an object inside the test and destroy it right after. This way the other tests can keep using the shared object and the test which call destroy is truly independent

Reply:
> This way the other tests can keep using the shared object

The objects right now(blocked and blocker) are being created with `let` so they only exist for the duration of the test and don't persist in between which means none of these tests should have shared objects so we should be good here.

Reply:
oh, right! :D